airflow/utils/cli_action_loggers.py

```diff
@@ -18,7 +18,7 @@
 #
 """
 An Action Logger module. Singleton pattern has been applied into this module
-so that registered callbacks can be used all through the same python process.
+so that registered callbacks can be used all through the same Python process.
```

Comment:
(Strictly speaking this is correct lowercase -- it's talking about the executable)